db/models.py

```diff
@@ -50,3 +60,57 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=CASCADE)
```

Comment:
This line of code violates the DRY (Don't Repeat Yourself) principle. The `CASCADE` import is used only once. It's better to use `models.CASCADE` directly for the `on_delete` parameter.